packages/frontend-shared/src/utils/isAllowedFeature.ts

```diff
@@ -47,6 +51,7 @@ export const isAllowedFeature = (
     navCiPromptAutoOpened: promptsShown.ci1,
     loginModalRecordPromptShown: promptsShown.loginModalRecord,
     latestSmartBannerShown: latestBannerShownTime,
+    debugPageRelease: new Date('2022-12-20T00:00:00').getTime(),
```

Comment:
What about exposing this value so it can be used in the tests instead of hardcoding in two places?

Reply:
Addressed in [4c21f5f](https://github.com/cypress-io/cypress/pull/25152/commits/4c21f5f4d005a19a9dbe52ffac3101f9bfeb513f)